src/java/org/apache/cassandra/auth/RoleOptions.java

```diff
@@ -89,6 +90,15 @@ public boolean isEmpty()
         return Optional.fromNullable((String)options.get(IRoleManager.Option.PASSWORD));
     }
 
+    /**
+     * Return the string value of the hashed password option
+     * @return hashed password option value
+     */
+    public Optional<String> getHashedPassword()
+    {
+        return Optional.fromNullable((String)options.get(IRoleManager.Option.HASHED_PASSWORD));
```

Comment:
Nit: missed whitespace
```suggestion
        return Optional.fromNullable((String) options.get(IRoleManager.Option.HASHED_PASSWORD));
```